go/libraries/doltcore/sqle/enginetest/dolt_engine_test.go

```diff
@@ -42,7 +42,7 @@ func limitTestQueriesTo(queries ...string) {
 }
 
 func TestQueries(t *testing.T) {
-	// limitTestQueriesTo(...) // whitelist queries you want run.
+	limitTestQueriesTo("SELECT i FROM niltable WHERE i2 IS NOT NULL ORDER BY 1") // whitelist queries you want run.
```

Comment:
this helper function is hiding test failures in the engine tests

Reply:
shit. didn't mean to check this in.